src/components/code-embed/code-embed.module.scss

```
@@ -2,34 +2,34 @@
 	--code-embed_address_padding: var(--spc-3x);
-	--code-embed_address_color: var(--foreground_emphasis-medium);
-	--code-embed_address_input_color: var(--foreground_emphasis-high);
-	--code-embed_address_outline_color: var(--surface_primary_emphasis-low);
-	--code-embed_address_outline_color_hovered: var(--foreground_emphasis-medium);
-	--code-embed_address_outline_color_focused: var(--secondary_default);
+	--code-embed_address_color: var(--foreground_medium);
+	--code-embed_address_input_color: var(--foreground_high);
+	--code-embed_address_outline_color: var(--tint_primary_low);
+	--code-embed_address_outline_color_hovered: var(--foreground_medium);
+	--code-embed_address_outline_color_focused: var(--focus-outline_secondary);
 	--code-embed_address_outline_width: var(--border-width_s);
```

Stylelint (17.14.0)

[error] Duplicate custom property "--code-embed_address_outline_width" (declaration-block-no-duplicate-custom-properties)

(declaration-block-no-duplicate-custom-properties)